### ISSUE TITLE
FOREACH clause example failed. [NEED HELP] 

### ISSUE THREAD

**Issue by liketemple** [memgraph/memgraph#1101]
**Memgraph version**
docker image 2.9 platform

**Environment**
ubuntu server 20.04

**Describe the bug**
FOREACH clause example from https://memgraph.com/docs/cypher-manual/extension-clauses failed, seems foreach not a correct clause

**To Reproduce**
1. Run the following query 'FOREACH ( i IN [1, 2, 3] | CREATE (n {id : i}) )'
got this:
Query failed: line 1:1 mismatched input 'foreach' expecting {CHECK, CLEAR, DENY, DROP, DUMP, FREE, GRANT, LOAD, LOCK, REGISTER, REVOKE, START, STOP, UNLOCK, CALL, CREATE, DELETE, DETACH, EXPLAIN, MATCH, MERGE, OPTIONAL, PROFILE, REMOVE, RETURN, SET, SHOW, UNWIND, WITH}

**Comment by gitbuda** [comment]
TLDR: I haven't been able to reproduce -> could you try again? 👀 

Hi @liketemple! The query passed for me by using the plain memgraph Docker image -> https://hub.docker.com/layers/memgraph/memgraph/2.9.0/images/sha256-cc8438dbbfeb00a531056ec5c87502ae53e83fd029f4f55a78742647f0844372?context=explore

![Screenshot 2023-07-25 at 1 04 07 PM](https://github.com/memgraph/memgraph/assets/4950251/9f17e0a0-65c8-4aed-a82c-b6c88ffdd0d5)

testing with the platform image -> that also works -> so, have no idea what is going on 🤔 

![Screenshot 2023-07-25 at 1 11 32 PM](https://github.com/memgraph/memgraph/assets/4950251/9592086e-bca8-4c2e-abcd-727cc80f2899)

**Comment by liketemple** [comment]
<img width="1154" alt="截屏2023-07-25 20 37 15" src="https://github.com/memgraph/memgraph/assets/10526411/7bd78fbf-d764-4662-a771-0d85897c0343">
I am new from neo4j, so I want to test as many clauses as possible, like foreach unwind.

**Comment by gitbuda** [comment]
Very weird this error is coming from memgraph 100% but, a few contradictions:
* this screen shot seems like a super old version of lab -> I've checked in the latest platform and there is the latest lab used
* in the list returned as an error there is no `FOREACH` -> can you double check which version of Memgraph are you using

seems like you are using some very old versions of Memgraph and Lab somehow 🤔

**Comment by liketemple** [comment]
I tried again, and found this:
When I pull the image from ubuntu server 20.04 automatically pull the image using Lab with version 1.3.5
![1690334438163](https://github.com/memgraph/memgraph/assets/10526411/25492f5c-2083-49be-bbcb-8e73b364d674)
![1690334384688](https://github.com/memgraph/memgraph/assets/10526411/dc801134-958a-4100-abd8-95ce6479fd33)

But when I pull the image from my mbp M2, the image is using Lab with version 2.7.1
![1690334726773](https://github.com/memgraph/memgraph/assets/10526411/24598ebc-6181-4f90-a65f-4260f068bbf0)

**Comment by liketemple** [comment]
And I tried image with tag "2.9.0-memgraph2.9.0-lab2.7.1-mage1.8" on ubuntu server 20.04, the lab version is also 1.3.5.
![image](https://github.com/memgraph/memgraph/assets/10526411/774e3acd-565e-4876-ba6a-6e3ea131150f)
![image](https://github.com/memgraph/memgraph/assets/10526411/17d6ae90-7f5e-4e17-ac67-1189caff4aa0)

**Comment by liketemple** [comment]
I pull the memgraph/memgraph image instead of platform, and use the standalone memgraph lab to connect. works fine on the ubuntu server 20.04 now.

**Comment by gitbuda** [comment]
@liketemple great! This seems to be resolved -> I'll close this issue, feel free to reopen if something is not right 😃
